### ISSUE TITLE
Automatically notify users of automatic role changes

### ISSUE THREAD

**Issue by adelikat** [TASVideos/tasvideos#147]
Right now we have to remember to notify users after a role page, and link them useful pages such as guidelines and text formatting rules.

Instead on Role change, we could generate a private message from TVA with all the necessary content.
Some todos ncecessary for this:
1) Remove System/UserEditorRole page, we no longer need to remind them
2) Replace with a system page that says that changes will notify the user
3) Create generic PM messaging system, we want to send users TVA messages for other things in the future too

**Comment by adelikat** [comment]
Every time I think about this, I think about the scenario where the editor makes a mistake and then corrects it, now we have spammed the user with messages.  Even more problematic, what if the intent was to make them an editor and you misclick banned user? and it notifies them they have been banned.

I think the idea is nice but better is a conversation to the user when changing their roles, and an automatic solution is not a sufficient alternative to this conversation.

Where this could add value is for automatic role upgrades such as Experienced Forum User and Published Author where the assignment is automatic.  For this reason this is still valuable and I updated the title accordingly.
